code/game/mecha/working/ripley.dm

```diff
@@ -54,22 +55,30 @@
 
 /obj/mecha/working/ripley/update_desc()
 	. = ..()
-	if(!hides) // Just in case if hides are somehow removed
+	if(!hides && !plates) // Just in case if armour is somehow removed
 		desc = initial(desc)
 		return
 	if(hides == 3)
 		desc = "Autonomous Power Loader Unit. It's wearing a fearsome carapace entirely composed of goliath hide plates - its pilot must be an experienced monster hunter."
 	else
 		desc = "Autonomous Power Loader Unit. Its armour is enhanced with some goliath hide plates."
+	if(plates == 3)
```

Comment:
please turn this magic number into a define

Reply:
Also do that for the hides in line 61